### PR TITLE
Advancement Fee More Info Links

### PR DESCRIPTION
**Pull request by funkSoRubber**
Create a help icon next to the advancement fee checkbox, and made the "advancement fee" text hyperlink to the WSU Foundation fees and disclosures page. We used flow-root display to show the link in firefox and chrome, and apply clearfix for other browsers. This still needs to be tested in Edge and Safari, as it couldn't be tested with these browsers in development.

### CODE REVIEW

File: includes/wsuwp-shortcode-fundselector-utils.js [hunk 1]
```diff
@@ -265,9 +265,15 @@ var EPSILON = 0;
 				};
 				
 				jQuery("#advFeeAmount").text(_.template(wpData.adv_fee_message)({advFeeAmount: advFeeDecimal.toFixed(2) }));
+				jQuery("#advFeeAmount").html(jQuery("#advFeeAmount").html().replace("Advancement Fee", '<a href="https://foundation.wsu.edu/about/fee-disclosure/" target="_blank">Advancement Fee</a>'));
```
Comment (blairlierman) [inline includes/wsuwp-shortcode-fundselector-utils.js]: This should also include the `rel="noopener noreferrer"` attribute in addition to `target="_blank"` for both [security](https://github.com/rwjblue/ember-template-lint#link-rel-noopener) and [performance](https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/) reasons.

File: includes/wsuwp-shortcode-fundselector-utils.js [hunk 2]
```diff
@@ -265,9 +265,15 @@ var EPSILON = 0;
...
+				jQuery("#advFeeAmount").html(jQuery("#advFeeAmount").html() + '<a id="moreAdvFeeInfo" href="https://foundation.wsu.edu/about/fee-disclosure/" target="_blank"></a>');
```
Comment (blairlierman) [inline includes/wsuwp-shortcode-fundselector-utils.js]: This should also include the `rel="noopener noreferrer"` attribute in addition to `target="_blank"` for both [security](https://github.com/rwjblue/ember-template-lint#link-rel-noopener) and [performance](https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/) reasons.

Comment (blairlierman) [inline includes/wsuwp-shortcode-fundselector-utils.js]: The updated link is now https://foundation.wsu.edu/fees/, based on feedback from Trevor.

File: css/wsuwp-plugin-idonate.css [hunk 3]
```diff
@@ -730,6 +730,18 @@
...
 }
+#fundSelectionForm .help-icon a#moreAdvFeeInfo {
+  display: inline-block;
+  right: 0;
+  top: 0;
+  width: 15px;
+  height: 15px;
+  padding: 0;
+  background: url(../images/ico-help.png) no-repeat center center;
+  background-size: 15px;
```
Comment (blairlierman) [inline css/wsuwp-plugin-idonate.css]: If we are going to use a 15px image, we should have a resized version of the icon as well. The current icon doesn't scale very well in IE at at all. 
fund030

Comment (funkSoRubber) [inline css/wsuwp-plugin-idonate.css]: I've resized the image and it looks much better on IE now
